contracts/contracts/ramps/wise/interfaces/ITLSData.sol

```diff
@@ -0,0 +1,11 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.18;
+
+interface ITLSData {
+    struct TLSParams {
```

Comment:
This is a bit opinionated but I would break TLSParams to endpoint and headers. Or rather, endpoint, host and headers. Endpoint and host should be constant for a given processor. Where as, headers might change. And data would definitely change.

Reply:
What do you consider headers to be? Haven't seen that term in any of the calldata. I mainly want to just group all the data required for validating TLSN transactions together (this is separate from any business logic)